### PR TITLE
Merge Release/6.0.1 to dev

### PR DESCRIPTION
**Pull request by rpdome**

### CODE REVIEW

**Review by tanmaymanolkar1 — APPROVED** [review]

**Review by p3dr0rv — APPROVED** [review]

File: common/build.gradle [hunk 1]
```diff
@@ -28,7 +28,7 @@ codeCoverageReport {
     coverage.enabled = enableCodeCoverage
 }
 
-def common4jVersion = "0.0.+"
```
Comment (iamgusain) [inline common/build.gradle:31]: please revert this

**Review by iamgusain — CHANGES_REQUESTED** [review]

File: common/build.gradle (hunk 1 above)
Comment (iamgusain) [reply common/build.gradle:31]: in `dev` we want to keep the dependencies(common4j, broker4j, common) to 0.0.+ to be able to be consumed by daily dev pipeline..

in `release/*` we change these to specific versions being consumed

**Review by iamgusain — APPROVED** [review]
Can you please also add that note to our release onenote.
Thanks

**Comment by rpdome** [comment]
> Can you please also add that note to our release onenote. Thanks

[done](https://microsoft.sharepoint.com/teams/ADAL/_layouts/15/Doc.aspx?sourcedoc={0339bdd6-b5e5-4d94-b4c9-ea47127f5023}&action=edit&wd=target%28Platforms%2FAndroid.one%7C28086b2a-f539-bc47-b229-1610fc61c903%2FMonthly%20release%20checklist%7C4353c4a1-b211-d844-9f58-a0e737d0c60a%2F%29&wdorigin=703)

**Pull request merged**